lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -0,0 +1,80 @@
+defmodule ArchEthicWeb.FaucetController do
+  @moduledoc false
+
+  alias ArchEthic.TransactionChain.{
+    Transaction,
+    TransactionData,
+    TransactionData.Ledger,
+    TransactionData.UCOLedger
+  }
+
+  alias ArchEthic.Crypto
+
+  use ArchEthicWeb, :controller
+
+  def index(conn, __params) do
+    conn
+    |> put_resp_header("cache-control", "no-cache, no-store, must-revalidate")
+    |> put_resp_header("pragma", "no-cache")
+    |> put_resp_header("expires", "0")
+    |> render("index.html", address: "")
+  end
+
+  def create_transfer(conn, %{"address" => address}) do
+    case transfer(address, Crypto.default_curve()) do
+      :ok ->
+        conn
+        |> put_flash(:info, "Transferred successfully")
+        |> redirect(to: "/faucet")
+
+      {:error, _} ->
+        conn
+        |> put_flash(:error, "Unable to transfer")
+        |> render("index.html", address: address)
+    end
+  end
+
+  defp transfer(
+         address,
+         curve
```

Comment:
Why do you want to use a curve because faucet transactions will be the same everytime.

Reply:
I was using this as a utility function for debugging.

Reply:
So you this one as default parameter, ie: ```def transfer(address, curve \\ Crypto.default_curve())```

Reply:
Yes I did that in my earlier version when it was not a private function. Should I make it public now?

Reply:
No need to make it public